tests/ContentServiceTest.php

```diff
@@ -103,7 +129,42 @@ public function it_adds_an_item() : void
 
         $response = $this->handle(Request::create('/items/new-article/versions/1'));
 
-        $this->assertXmlStringEqualsXmlString($expected, $response->getContent());
+        $this->assertXmlStringEqualsXmlString(
+            <<<XML
+<item xmlns="http://libero.pub">
+    <meta>
+        <id>new-article</id>
+        <service>articles</service>
+    </meta>
+    <article xmlns="http://jats.nlm.nih.gov" xmlns:xlink="http://www.w3.org/1999/xlink"
+        xml:base="https://www.example.com/new-article/">
```

Comment:
`@xml:base` should be removed, would need to check that nothing uses it anymore. Ignoring for now.